Source/WebInspectorUI/UserInterface/Views/SpreadsheetStyleProperty.js

```diff
@@ -940,7 +950,7 @@ WI.SpreadsheetStyleProperty = class SpreadsheetStyleProperty extends WI.Object
 
                     let text = rawTokens.reduce((accumulator, token) => accumulator + token.value, "");
                     if (this._property.ownerStyle.nodeStyles.computedStyle.resolveVariableValue(text))
-                        newTokens.push(this._createInlineSwatch(WI.InlineSwatch.Type.Variable, contents));
+                        newTokens.push(this._createInlineSwatch(WI.InlineSwatch.Type.Variable, contents, text));
```

Comment:
is providing this actually necessary?  it seems like it's overridden inside `_createInlineSwatch` by the arrow function on :614

Reply:
I actually had a mistake within this code, we need to pass in text because it's what holds the values like --variable-name. Then later I need to do another check before all variable types are over written by the function you're referring to on :614. I'm going to check to make sure that whatever text came through isn't being utilized within the color palette. I've fixed it in the new patch, but for reference